api/src/pcapi/core/providers/allocine.py

```diff
@@ -141,6 +142,14 @@ def get_most_recent_release_date(releases: list[allocine_serializers.AllocineMov
     return sorted_releases[0] if sorted_releases else None
 
 
+def get_certificate(releases: list[allocine_serializers.AllocineMovieRelease]) -> str | None:
+    sorted_certificates = sorted(
+        [release.certificate for release in releases if release.certificate and release.certificate],
```

Comment:
Il y a répétition de la condition
`if release.certificate and release.certificate`
Oubli ?

Reply:
il semblerait en effet x)